Makefile

```diff
@@ -112,10 +112,12 @@ build_tools:
 	$(HOSTGO) build -o ./tools/custom_builder/custom_builder$(EXEEXT) ./tools/custom_builder
 	$(HOSTGO) build -o ./tools/license_checker/license_checker$(EXEEXT) ./tools/license_checker
 	$(HOSTGO) build -o ./tools/readme_config_includer/generator$(EXEEXT) ./tools/readme_config_includer/generator.go
+	$(HOSTGO) build -o ./tools/config_includer/generator$(EXEEXT) ./tools/config_includer/generator.go
 	$(HOSTGO) build -o ./tools/readme_linter/readme_linter$(EXEEXT) ./tools/readme_linter
 
 embed_readme_%:
-	go generate -run="readme_config_includer/generator$$" ./plugins/$*/...
+	go generate -run="tools/config_includer/generator\s?.*$$" ./plugins/$*/...
+	go generate -run="tools/readme_config_includer/generator\s?.*$$" ./plugins/$*/...
```

Comment:
Is there a scenario where we would ever want to run config_includer without running readme_config_includer as well? If not, should these be combined into one? It would also help with the not-so-clear naming and avoid any ordering issues

Reply:
I think yes. You might e.g. want to include the TLS config from `plugins/common/http` and there you don't have a README necessarily...

Reply:
Fixed.